ui/src/main/resources/PhenoTips/VariantDisplay.xml

```diff
@@ -167,14 +167,15 @@
             this.optionsButton.update(new Element('a', {'class' : 'button', 'href' : '#'}).insert(this.Strings["hideOptionsButton"])).stopObserving('click', this.showOptions).observe('click', this.hideOptions.bind(this));
             
             var numberOfGenesInput = new Element('select', {'id' : 'variant-display-number-of-genes'});
-            for (var i = 1; i &lt;= 20; i++) {
+            var maxGenes = ("$xwiki.hasAccessLevel('edit')" == "true") ? 500 : 10;
+            for (var i = 1; i &lt;= maxGenes; i++) {
```

Comment:
Having 500 options is not good. Nobody wants exactly 432 genes, use an increasing step: `1 2 3 4 5 10 20 ... 90 100 150 200 300 500`.